frontend/pages/club/[club]/renew.tsx

```diff
@@ -245,6 +245,14 @@ const RenewPage = ({
 
   const nextStep = () => {
     setStep(step + 1)
+    if (step === 3) {
+      doApiRequest(`/clubs/${club.code}/?format=json`, {
```

Comment:
@cphalen I modified this to use an existing endpoint instead of creating a new endpoint!